src/lib/components/NewLayeredMap/layers/L.imageWebGLOverlay.js

```diff
@@ -111,9 +122,14 @@ L.ImageWebGLOverlay = L.Layer.extend({
     updateOptions: function(options) {
         const promisesToWaitFor = [];
 
+        if(this.options.colorScale) {
+            const newColorScale = this.reUseColorscale(options)
+            options.colorScale = newColorScale
```

Comment:
This removes the possibility to remove the colorScale.  You should instead merge with the other colorScale-checks in this function:

```
if(!options.colorScale) {
            this._colormapUrl = null;
} else {
            if(options.colorScale instanceof Object && this.options.colorScale instanceof Object) {
                  options.colorScale = Object.assign(this.options.colorScale, options.ColorScale)
            }
            this._initColormap();
}
```
Something like that. If may have to move the if-statements around in the function, but something like above is enough.